roles-in-tech/dev-ops.md

```diff
@@ -17,75 +13,75 @@ Here are links to lessons that should be completed before this lesson.
 
-### Lesson
+Some Continuos Delivery Tools are
+- [JBoss](http://www.jboss.org/) owned by Red Hat is a web application server which is fully integrated in order to host JAVA driven applications (applications developed using Java EE platform).
+- Apache [TOMCAT](http://tomcat.apache.org/), also referred as Tomcat server is developed by ASF (Apache Software Foundation). It includes the integration of various Java specifications like Java Servlet, Java EE, Java EL, Web socket, server pages, Java expressions etc. that build a pure environment for Java code to run.
 
-Here's text about introducing something and how it works.
+Other DevOps Tools are 
+- [Docker](https://www.docker.com/)is an instrument that helps with packaging code into self-contained units, i.e. containers.
+- [Git](https://git-scm.com/)is a Version Control System with a repository for source code management that enables working online and offline.
 
-- [Be sure to link the slideshow, video,](google.com)
-- [or other materials you expect them to follow as part of the lesson.](google.com)
+- [Nagios](https://www.nagios.org/) is an infrastructure monitoring tool that presents analytics in visual reports
+- [Selenium](https://www.seleniumhq.org/) is an automated browser that allows QA teams to write scripts and test web products. It’s compatible with eight popular programming languages.
 
-Build on the first information. Have apprentices guess things, do an activity, etc.
+- *For better understanding of **DevOps** refer to this slide [Roles in Tech:DevOps](https://docs.google.com/presentation/d/1GaoHCdvQ2UplDjapp2wG-VRsxlqtzFqIpOtIqvKNYOo/edit?usp=sharing)*
 
-Make sure to mention these things:
+### Lesson
 
-- Things
-	- This is a sub-thing
-- More things
-- Even more things
-- Even more things
+**_DevOps_** stands for development and operations. It’s a practice that aims at merging development, quality assurance, and operations (deployment and integration) into a single, continuous set of processes. This methodology is a natural extension for Agile and continuous delivery approaches.
 
+For Better Understanding of *DevOps* , one should do continuous unit testing after deploying it on a suitable platform.
 
 ### Common Mistakes / Misconceptions
 
-List things that apprentices might not realize, might assume at first, or should avoid.
-
-- Example
-- Example
+While many industry leaders have successfully deployed DevOps to achieve faster speed-to-market with higher quality software and a first-class customer experience, there continues to be some fundamental misconceptions about what DevOps really means.
 
+- **DevOps is a set of automated tools.**
+- **DevOps procedures change from project to project.**: 
 
 ### Guided Practice
 
-Have the apprentices work with you as you do something step-by-step.  This can also be fulfilled by a detailed tutorial intended for beginners.
-
+[DevOps lessons from Edureka](https://www.edureka.co/devops)
```

Comment:
Which lesson(s) should they do if they have 5-15 minutes to try it? Describe why this is a good resource.

Reply:
I have changed my PR according to your requirements @alodahl and @manaswinidas .